serverless/index-serverless-project-settings.asciidoc

```diff
@@ -1,8 +1,10 @@
 [[project-and-management-settings]]
 == Project and management settings
 
-include::./pages/project-and-management-settings.asciidoc[leveloffset=+2]
 
+
+include::./pages/project-and-management-settings.asciidoc[leveloffset=+2]
+include::./index-serverless-devtools.asciidoc[leveloffset=+2]
```

Comment:
What if we get rid of this + set up a redirect to https://www.elastic.co/guide/en/kibana/current/devtools-kibana.html instead?